### ISSUE TITLE
Non-technical how-to for calling the callback API via Postman

### ISSUE THREAD

**Issue by baltes** [department-of-veterans-affairs/va.gov-team#87682]
As a member of the IVC forms team who's not an engineer, I need a guide for getting Postman set up to call the callback API and see the responses.

Acceptance criteria
- I have guidance for getting an authentication token that I can execute without intervention
- I have guidance for how to set up Postman, or import a workspace, so that I can call the API with different payloads and receive a range of responses appropriate for that payload.

**Comment by balexandr** [comment]
@baltes  After speaking with the VSP Identity team on a couple of occasions, it seems like there isn't a non-technical/less technical way to retrieve the JWT assertion for the Postman call to our endpoint. I've tried to set up Postman pre-scripts to no avail. I've attempted to use jwt.io which can grant a JWT assertion but you still need the private key and the ability to use some code to generate an expiration date and JTI for the payload.

I think the best we can do is make sure the engineers on the team know how to achieve the JWT assertion so it doesn't rest solely on one person. 

<img width="1018" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBMy9GQlE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--63cd4c599f71bd5e75b787dd735e37b17dc262df/Screenshot%202024-07-09%20at%2011.13.29%E2%80%AFAM.png" alt="Screenshot 2024-07-09 at 11.13.29 AM.png" />

**Comment by balexandr** [comment]
@baltes  After speaking with the VSP Identity team on a couple of occasions, it seems like there isn't a non-technical/less technical way to retrieve the JWT assertion for the Postman call to our endpoint at this point. I've tried to set up Postman pre-scripts to no avail. I've attempted to use jwt.io which can grant a JWT assertion but you still need the private key and the ability to use some code to generate an expiration date and JTI for the payload.

I think the best we can do is make sure the engineers on the team know how to achieve the JWT assertion so it doesn't rest solely on one person. 

<img width="1018" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBMy9GQlE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--63cd4c599f71bd5e75b787dd735e37b17dc262df/Screenshot%202024-07-09%20at%2011.13.29%E2%80%AFAM.png" alt="Screenshot 2024-07-09 at 11.13.29 AM.png" />

**Comment by balexandr** [comment]
@baltes Seems like VSP Identity figured out a way through Postman to get the STS JWT Token. Here is their documentation:
https://github.com/department-of-veterans-affairs/va.gov-team/blob/707d8afd0e44447d73fff86894154966523a7005/products/identity/Products/Sign-In%20Service/Engineering%20Docs/Authentication%20Types/Service%20Account%20Auth%20(STS)/postman/postman.md

I tested this out and it works. I exported the collection below. Whoever uses it needs to copy and paste their STAGING PRIVATE KEY and running the `token` will output the FWT token and paste that into the status update staging call auth and it should work.
[VA.gov Identity - PEGA Service Account Auth (STS).postman_collection.json](https://github.com/user-attachments/files/16238985/VA.gov.Identity.-.PEGA.Service.Account.Auth.STS.postman_collection.json)
